analysis_options.yaml

```diff
@@ -141,9 +141,6 @@ linter:
     # Style: Sort unnamed constructor declarations first
     - sort_unnamed_constructors_first
 
-    # Style: Place the super() call last in a constructor initialization list.
-    - super_goes_last
```

Comment:
Do we have a template for these options these days? Or like, how did you decide what changes to make here?

Reply:
Dart analysis just flagged them as analysis options that are no longer used? It didn't offer any alternative or new pattern, just said they don't do anything anymore.

Reply:
> 'super_goes_last' is a deprecated lint rule and should not be useddart(deprecated_lint_hint)